From 60b13e3b5f5d3ada9a4f6456d75bac9a452094cc Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 15 Aug 2013 16:50:14 +0800
Subject: [PATCH] ENGR00275202 Mms: fix one NPE in Mms app

if not declare support mobile_mms networkAttributes but ethernet in config.xml
it will trigger this null pointer exception.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 src/com/android/mms/transaction/MmsSystemEventReceiver.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/com/android/mms/transaction/MmsSystemEventReceiver.java b/src/com/android/mms/transaction/MmsSystemEventReceiver.java
index e152b3a..77dd14d 100644
--- a/src/com/android/mms/transaction/MmsSystemEventReceiver.java
+++ b/src/com/android/mms/transaction/MmsSystemEventReceiver.java
@@ -75,6 +75,8 @@ public class MmsSystemEventReceiver extends BroadcastReceiver {
             }
             NetworkInfo mmsNetworkInfo = mConnMgr
                     .getNetworkInfo(ConnectivityManager.TYPE_MOBILE_MMS);
+            if (mmsNetworkInfo == null)
+                return;
             boolean available = mmsNetworkInfo.isAvailable();
             boolean isConnected = mmsNetworkInfo.isConnected();
 
-- 
1.8.0

